### ISSUE TITLE
Create or find a Dequeue slideshow

### ISSUE THREAD

**Issue by alodahl** [Techtonica/curriculum#537]
- Base your slideshow on this topic outline: https://github.com/Techtonica/curriculum/blob/master/data-structures/deque.md

- [x] Make or find a slideshow about dequeues.  If you're making one, save it in this file:
https://docs.google.com/presentation/d/10zp71xtt0j0a7CaSFXru6abTFKIPiBxwhlbrlKfyH9A/edit?usp=sharing

For FAQs on contributing, see https://github.com/Techtonica/curriculum

**Comment by bobbylcraig** [comment]
👋 Hello again! Looking into this one now.

**Comment by bobbylcraig** [comment]
@alodahl I think I've got a good basis in the current form. I can add more implementation details (although I feel a video/code might be more helpful) or add more about the A-Steal algorithm (although that might be a bit of overkill 🤷‍♂️). Otherwise, it should be ready for proofreading.

FWIW, I notice that priority queues aren't in the data structures list and I'd imagine they'd be much more valuable than deques. I'm happy to work on that too. Let me know :)

ALSO ALSO (I swear this is the last one 😄), not sure how people feel about gifs in presentations, but I added one 😬. It's easily removed if it's a problem.

**Comment by alodahl** [comment]
@bobbylcraig This looks great!! If you have time and the interest to add a video and the A-Steal algorithm, that would be fantastic. We like gifs!

Queues are the topic apprentices are learning right before, but are you saying priority queues are more specific?  Someone is working on the queues slides, but would you like to take a look after she's done?

**Comment by bobbylcraig** [comment]
Yeah, I think a PQ and/or MPQ addendum would be kind of fun :)

Also, I opened a PR (https://github.com/Techtonica/curriculum/pull/552) for the A-Steal Algorithm info. Searched and couldn't find a great video––scheduling algorithms aren't always extremely intuitive. I included an academic paper that could be additional reading if someone felt so inclined.

With that, I think this issue should be good to go :)
